source/Engine/Application.cpp

```diff
@@ -654,6 +656,11 @@ void Application::Restart() {
 void Application::LoadVideoSettings() {
 	bool vsyncEnabled;
 	Application::Settings->GetBool("display", "vsync", &Graphics::VsyncEnabled);
+	Application::Settings->GetInteger("display", "frameSkip", &Application::FrameSkip);
```

Comment:
Minor nitpick: `GetInteger` returns `false` if the field couldn't be converted into an integer, and `true` if it could be.

But the side effects of not checking for that aren't catastrophic in this situation, so it's non-blocking.